.gitignore

```diff
@@ -45,4 +45,6 @@ Temporary Items
 .apdisk
 
 
-# End of https://www.gitignore.io/api/go,osx
\ No newline at end of file
+# End of https://www.gitignore.io/api/go,osxCerealNotes
+
+CerealNotes
```

Comment:
Is this popping up in the home directory? And not your bin directory?

Reply:
Ya its cause i did go build install of go install. We want to use go install for sure. I'm not sure i understand why its like that, but i don't have a strong feeling about it

Reply:
so we planning on removing this?

Reply:
yes